docs/DeveloperGuide.md

```diff
@@ -215,6 +216,40 @@ Note: Deliveries are all considered unique. Reason being the same person can mak
 <div markdown="span" class="alert alert-info">:information_source: **For example:**`John` with the address `Choa Chu Kang Block 259` is able to make multiple orders before his previous deliveries are fulfilled.
 </div>
 
+### Editing Items and Delivery
+OneShelf is capable of editing the current items and deliveries in the list.
+Editing Items and Deliveries both are done similarly which will be illustrated below.
+
+<div markdown="span" class="alert alert-info">:information_source: **For this purpose:**
+`ItemEditCommand` and `DeliveryEditCommand` will be referred as `EditCommand`, similarly to `EditDescriptor` and `EditCommandParser`
+</div>
+
+The following is a class diagram for the `EditCommand`. It takes in an `Index` and a `EditDescriptor` class respectively, which contains of the changes to be made.
+
+![EditClassDiagram](images/EditClassDiagram.png) <br>
```

Comment:
The class diagram has commands, shouldnt it be blue in colour?

Reply:
good catch. my bad on this